nginx-php-moodle/bin/watch_certificate_dir.sh

```diff
@@ -22,7 +22,7 @@ source /usr/local/bin/shtdlib.sh
 moodle_cert='/etc/nginx/ssl/moodle.crt'
 moodle_key='/etc/nginx/ssl/moodle.key'
 
-echo "Watching ${CERT_DIR} for updated certs..."
+echo "Watching '${CERT_DIR}' for updated certs..."
```

Comment:
Does the value still get interpolated if it has the single quotes around?

Reply:
Yes, the outside double quotes is what matters.

Reply:
Cool TIL